src/Elliptic/DiscontinuousGalerkin/Actions/ApplyOperator.hpp

```diff
@@ -555,8 +558,23 @@ struct ImposeInhomogeneousBoundaryConditionsOnSource<
       db::DataBox<DbTags>& box,
       const tuples::TaggedTuple<InboxTags...>& /*inboxes*/,
       const Parallel::GlobalCache<Metavariables>& /*cache*/,
-      const ElementId<Dim>& element_id, const ActionList /*meta*/,
+      const ElementId<Dim>& /*element_id*/, const ActionList /*meta*/,
       const ParallelComponent* const /*meta*/) {
+    db::mutate_apply<ImposeInhomogeneousBoundaryConditionsOnSource>(
+        make_not_null(&box));
+    return {Parallel::AlgorithmExecution::Continue, std::nullopt};
+  }
```

Comment:
Optionally, you can remove this function and use `Actions::MutateApply` in the executable.

Reply:
Is there an advantage to that? I guess it makes the action a bit simpler by removing the iterable action interface.

Reply:
Right, the advantage is just that there's less code.